app/src/organisms/ODD/QuickTransferFlow/utils/generateQuickTransferArgs.ts

```diff
@@ -96,18 +98,28 @@ function getInvariantContextAndRobotState(
       },
     }
   }
+  const sourceDisplayCategory =
+    quickTransferState.source.metadata.displayCategory
+  const destDisplayCategory =
+    quickTransferState.destination !== 'source'
+      ? quickTransferState.destination.metadata.displayCategory
+      : 'unknown'
```

Comment:
This means they've selected to use only one labware for their transfer, so I think this makes more sense:
```suggestion
      :  sourceDisplayCategory
```